core/crypto/signatures/p256/ecdsa_p256_test.go

```diff
@@ -230,7 +230,7 @@ func flipBit(a []byte, pos uint) []byte {
 
 func TestGeneratePEMs(t *testing.T) {
 	signatures.Rand = DevZero{}
-	skPEM, pkPEM, err := GeneratePEMs()
```

Comment:
Please keep this error, check it, and stop the test with `t.Fatalf` if it's not nil.